.github/workflows/test-compatibility.yml

```diff
@@ -40,10 +40,12 @@ jobs:
           export OPENSEARCH_VERSION=${{ matrix.entry.opensearch_version }}
           export SECURE_INTEGRATION=${{ matrix.secured }}
           make cluster.clean cluster.build cluster.start
+#          sleep 20
```

Comment:
Looks like you've commented this out instead of actually increasing timeout?

Reply:
Yes, I checked this case without timeout and there is success result. I think it was just a coincidence at the first time. So, it has nothing to do with timeout, I'm trying to dig further. But locally everything works fine for me with the version of the opensearch image the same as on the erroneous. So I'm at a loss as to why this is happening

Reply:
Yeap, I created a loop to test as you suggested. And there are two problems:
1. Status check does not work with a [secure connection (https)](https://github.com/zethuman/opensearch-go/actions/runs/4773852492/jobs/8487335276).
2. Also, there seems to be [some problem](https://github.com/zethuman/opensearch-go/actions/runs/4773852492/jobs/8487335428) with starting the server, as it stops responding to status requests and just closes the connection. 

I keep digging